test/config.js

```diff
@@ -16,10 +16,10 @@ var optionFixtures = [
   ['captureExceptions', 'CAPTURE_EXCEPTIONS', true],
   ['instrument', 'INSTRUMENT', true],
   ['asyncHooks', 'ASYNC_HOOKS', true],
-  ['sourceContextErrorAppFrames', 'SOURCE_CONTEXT_ERROR_APP_FRAMES', 5],
-  ['sourceContextErrorLibraryFrames', 'SOURCE_CONTEXT_ERROR_LIBRARY_FRAMES', 5],
-  ['sourceContextSpanAppFrames', 'SOURCE_CONTEXT_SPAN_APP_FRAMES', 5],
-  ['sourceContextSpanLibraryFrames', 'SOURCE_CONTEXT_SPAN_LIBRARY_FRAMES', 0],
+  ['sourceLinesErrorAppFrames', 'SOURCE_CONTEXT_ERROR_APP_FRAMES', 5],
+  ['sourceLinesErrorLibraryFrames', 'SOURCE_CONTEXT_ERROR_LIBRARY_FRAMES', 5],
+  ['sourceLinesSpanAppFrames', 'SOURCE_CONTEXT_SPAN_APP_FRAMES', 5],
+  ['sourceLinesSpanLibraryFrames', 'SOURCE_CONTEXT_SPAN_LIBRARY_FRAMES', 0],
```

Comment:
s/CONTEXT/LINES/

(also on the 3 lines above)

Reply:
fixed